concepts/observability-strategy/configurable-tracing/02-tracing-collectors.md

```diff
@@ -2,9 +2,32 @@
 
 ## Requirements
 
+The agent's responsibility is to collect all span data in the cluster (Istio, Serverless, Eventing, custom) and ship them in a configurable way to targets. The agent is not meant to be used as backend system, so ideally it is even stateless.
+
+- Collection
+  - Support OTLP push approach, ideally with service discovery. A lot of apps are based on the pull approach and should not be rewritten or operated with a sidecar only for transforming that approach.
+  - Support push-based model using OTLP.
```

Comment:
Isn't this identical to line 8, just in fewer words?
Is line 8 really about push approach, or did you mean `pull` approach there (which would match the 2nd sentence better)